cryptoinvestor/models/asset.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, id: str, name: str, symbol: str, is_crypto: bool):
         self.symbol = symbol
         self.is_crypto = is_crypto
         self.rates = {}
+        self.coin_status = 10000
+        self.bought = dict()
```

Comment:
What is this dict?

Reply:
It represent what crypto did you bought and amount of them. For ex. "btc":1

Reply:
And why is this in `asset` model. BTC is a asset, can assets buy another assets?